engine/src/main/java/org/terasology/logic/characters/CharacterMovementComponent.java

```diff
@@ -71,6 +71,8 @@
     // Distance since last footstep
     public float footstepDelta;
 
+    // Distance desired from ground
+    public float groundDistance;
```

Comment:
Ah, this is just a bit of code I didn't remove from rolling back HoverCam mode (which will keep the player a... player controlled distance from the ground). I'm guessing I'll put that in a hoverCam component?